v1/brokers/amqp/amqp.go

```diff
@@ -214,7 +217,7 @@ func (b *Broker) Publish(ctx context.Context, signature *tasks.Signature) error
 		amqp.Table(b.GetConfig().AMQP.QueueBindingArgs), // queue binding args
 	)
 	if err != nil {
-		return err
+		return errors.Wrap(err, "Failed to get a connection")
```

Comment:
consider adding `queue` to the error reply

Reply:
done